app/build.gradle

```diff
@@ -75,14 +75,17 @@ android {
             storePassword '123456'
         }
     }
-    compileSdk 28
+    compileSdk 34
     defaultConfig {
-        applicationId "com.raygun.raygun4androidsample"
+        applicationId "com.raygun.raygun4android"
         minSdkVersion 16
-        targetSdkVersion 28
+        targetSdkVersion 34
         versionCode = Long.valueOf(VERSION_CODE)
         versionName = "${VERSION_NAME}-sample"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+    }
+    lintOptions {
+        disable 'LogNotTimber','StringFormatInTimber', 'ThrowableNotAtBeginning', 'BinaryOperationInTimber', 'TimberArgCount', 'TimberArgTypes', 'TimberTagLength'
```

Comment:
Just to confirm - We are excluding these lint options for now?

Reply:
Yes, in the `:app` module. Because the sample app doesn't use Timber itself and we don't want the `:provider`'s Timber inclusion trigger lint warnings in the app that are not relevant. And it's a great sample of how to work around the toolchain's flow.